adapters/repos/db/storobj/storage_object.go

```diff
@@ -74,10 +64,8 @@ func FromBinary(data []byte) (*Object, error) {
 
 func (ko *Object) Class() schema.ClassName {
 	switch ko.Kind {
```

Comment:
Most of these switches that always only have a single case and the default could also be removed as part of the "remove kind.Object" follow-up issue.